packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -1084,6 +1084,11 @@ export class LocalFluidDataStoreContextBase extends FluidDataStoreContext {
 			0x150 /* "pkg should be available in local data store context" */,
 		);
 
+		const idRange = this.idCompressor?.takeNextCreationRange();
```

Comment:
Double check that this is the right place for this. The intent here is that a detached container will finalize all of its IdCreationRanges before generating a summary since it doesn't need to wait on consensus.